scilpy/tractanalysis/mrds_along_streamlines.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from scilpy.tractanalysis.grid_intersections import grid_intersections
+
+
+def mrds_metrics_along_streamlines(sft, mrds_pdds,
+                                   metrics, max_theta,
+                                   length_weighting):
+    """
+    Compute mean map for a given lobe-specific metric along streamlines.
+
+    Parameters
+    ----------
+    sft : StatefulTractogram
+        StatefulTractogram containing the streamlines needed.
+    mrds_pdds : ndarray (X, Y, Z, 3*N_TENSORS)
+        MRDS principal diffusion directions of the tensors
+    metrics : list of ndarray
+        Array of shape (X, Y, Z, N_TENSORS) containing the fixel-specific
+        metric of interest.
+    max_theta : float
+        Maximum angle in degrees between the fiber direction and the
+        MRDS principal diffusion direction.
+    length_weighting : bool
+        If True, will weigh the metric values according to segment lengths.
+    """
+
+    mrds_sum, weights = \
+        mrds_metric_sums_along_streamlines(sft, mrds_pdds,
+                                           metrics, max_theta,
+                                           length_weighting)
+
+    all_metric = mrds_sum[0]
+    for curr_metric in mrds_sum[1:]:
+        all_metric += curr_metric
```

Comment:
Hmm. Isn't it just `all_metric= np.sum(curr_metric, axis=0)`?

But it's only used for non-zero. Is there a possibility that the metrics will cancel each-other?
Something like that could be safer?:

`non_zeros = np.sum(np.non_zeros(curr_metric), axis=0)`

Reply:
I just add a absolute value so it does not cancel each-other, also this method mimics afd_along_streamlines.py.